stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -124,8 +124,10 @@ function define_default_editors()
         `$cmd $path`
     end
     # vim family
-    for (editors, wait) in  [[Any["vim", "vi", "nvim", "mvim"], true],
-                             [Any["\bgvim"],                    false]]
+    for (editors, wait) in [
+        [["vim", "vi", "nvim", "mvim"], true],
+        [["\bgvim"], false],
```

Comment:
While you are at it, shouldn't this be `r"\bgvim"` ?

Reply:
```suggestion
        [[r"\bgvim"], false],
```